src/global.c

```diff
@@ -273,8 +273,9 @@ static void init_once(void)
 {
 	if ((init_error = git_mutex_init(&git__mwindow_mutex)) != 0)
 		return;
-	pthread_key_create(&_tls_key, &cb__free_status);
 
+	if (!_tls_key)
+		pthread_key_create(&_tls_key, &cb__free_status);
```

Comment:
This is checking an opaque value for a particular value, which we should not be doing in any case.